omero/sysadmins/server-setup-examples.rst

```diff
@@ -1,6 +1,32 @@
 Example production server set-ups
 =================================
 
+CellNanOs (Center of Cellular Nanoanalytics), University of Osnabrück
+---------------------------------------------------------------------
+
+The OMERO server at 
+`CellNanOS <https://www.cellnanos.uni-osnabrueck.de/en/startpage.html>`_
+serves a community of 75-100 users and 17 microscope stations (13 different
+systems), producing 180-1360 GB of data per day. It is hosted on RedHat 7.3
+with data stored on an IBM GPFS file system.
+
+Hardware
+^^^^^^^^
+
+- Dell R630 running RedHat 7.3, 32 cores, 128 GB RAM
+- IBM GPFS storage, 6 TB SSDs, 178 TB SATA
+
+Network infrastructure
+^^^^^^^^^^^^^^^^^^^^^^
+
+1-10 GBit connection between microscope workstations and OMERO
+
+Backup/archive
+^^^^^^^^^^^^^^
+
+- IBM TSM 1,4 PT
```

Comment:
Is this simply "1.4" with continental decimal punctuation?

Reply:
I'm honestly not sure which is why I just copied the slide! Would 1.4 make sense in this context?

Reply:
Any ideas @kennethgillen? If nobody knows can just leave as is. :smiley:

Reply:
Kenny assumes this is 1.4 PB, pushing a fix

Reply:
@kennethgillen suggests 2 things: 
  - transform comma into dot
  - instead of ``PT`` use ``PB`` (petabytes)